mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -116,3 +116,131 @@
         - :RELEASE
         - :SNAPSHOT
         default_value: :VERSION_POLICY_UNSPECIFIED
+  - !ruby/object:Api::Type::Enum
+    name: 'mode'
+    min_version: beta
+    description: |-
+      The mode configures the repository to serve artifacts from different sources.
+    input: true
+    values:
+      - :STANDARD_REPOSITORY
+      - :VIRTUAL_REPOSITORY
+      - :REMOTE_REPOSITORY
+    default_value: :STANDARD_REPOSITORY
+  - !ruby/object:Api::Type::NestedObject
+    name: 'virtualRepositoryConfig'
```

Comment:
Does this field virtualRepositoryConfig need input: true ?

Reply:
No. `virtual_repository_config` is mutable.